modules/mysql/read_replica.tf

```diff
@@ -80,39 +53,32 @@ resource "google_sql_database_instance" "replicas" {
         }
       }
     }
-    authorized_gae_applications = var.authorized_gae_applications
 
-    crash_safe_replication = var.read_replica_crash_safe_replication
-    disk_autoresize        = var.read_replica_disk_autoresize
```

Comment:
What's the reason for removing this?

Reply:
re: `crash_safe_replication` its leftovers from the now removed 1st gen instances. 
From Provider docs:
> Remove any fields that are not applicable to Second-generation instances: * settings.crash_safe_replication * settings.replication_type * settings.authorized_gae_applications

https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/instances
ctrl+f for `First Generation instances` to see further differences

Reply:
Right, I meant the `read_replica_disk_autoresize` variable.

Reply:
That's been removed in favour of using the master instance `var.disk_autoresize` to help simplify things and avoid potential problems down the line of users just enabling it on the master and not replicas. If the master disk increases, then replicas should be the same.

The same approach has been taken for `var.read_replica_disk_size` where its completely removed in favour of `var.disk_size`.

Although i can see a edge case where users may want a replica to have a larger disk size vs master (and they aren't using autoresize/near capacity already) due to certain queries creating temp tables on disk etc. I'm not sure how likely that scenario is/trying to minimise object fields since we still can't default them. Opinion on having that `disk_size` replica option available?

Reply:
I'd suggest we leave the existing `read_replica_disk_size` and `read_replica_disk_autoresize` untouched (to cover those corner cases) but default to using the master's settings.

Reply:
Fixed via https://github.com/terraform-google-modules/terraform-google-sql-db/pull/117/commits/8c97fe90475652cae7b9f5607048167737a0a2fe